pallets/subtensor/src/macros/dispatches.rs

```diff
@@ -1201,5 +1198,17 @@ mod dispatches {
         ) -> DispatchResult {
             Self::do_set_subnet_identity(origin, netuid, subnet_name, github_repo, subnet_contact)
         }
+
+        /// User register a new subnetwork
+        #[pallet::call_index(79)]
+        #[pallet::weight((Weight::from_parts(157_000_000, 0)
+                .saturating_add(T::DbWeight::get().reads(16))
+                .saturating_add(T::DbWeight::get().writes(30)), DispatchClass::Operational, Pays::No))]
+        pub fn register_network_with_identity(
+            origin: OriginFor<T>,
+            identity: Option<SubnetIdentityOf>,
+        ) -> DispatchResult {
+            Self::user_add_network_with_identity(origin, identity)
```

Comment:
Similarly, I suggest putting the contents of this fn inline here.

Reply:
Same as above